docs/openapi3.yaml

```diff
@@ -1052,6 +1052,12 @@ components:
           type: string
           description: The name of the test.
           example: Order from Pet Store
+        file_url:
```

Comment:
1. need to add in swagger the api definition of GET /file/{{file_id}}
2. need to add to tests response definition in swagger to return file_id in the parameters

Reply:
Fixed